lgc/patch/Gfx9ConfigBuilder.cpp

```diff
@@ -1901,8 +1901,6 @@ void ConfigBuilder::buildPsRegConfig(ShaderStage shaderStage, T *pConfig) {
 
   for (unsigned i = 0; i < interpInfo->size(); ++i) {
     auto interpInfoElem = (*interpInfo)[i];
-    if (m_pipelineState->isUnlinked() && interpInfoElem.loc == InvalidFsInterpInfo.loc)
```

Comment:
1. Is this a problem only because you're disabling packing on a shader compile?
2. Is the same fix needed in Gfx6ConfigBuilder.cpp (for gfx6,7,8)?

Reply:
1) Yes, it is because we disable packing.  
2) Possibly I'll check.